docs/modules/release-notes/pages/all-releases/4_14.adoc

```diff
@@ -0,0 +1,117 @@
+
+In SSO settings, when the option _Enforce users to login to Kobiton only through SSO_ is turned on, there is now a new field *_Choose users who are allowed to login without SSO_*. Search for and add existing Kobiton users to this field to allow them to log in using Kobiton username and password.
+
+When _Enforce users to login to Kobiton only through SSO_ is turned on, the password of all existing users are cleared from database. Therefore, the users that are allowed to log in using Kobiton username and password must use *Forgot password* to create a new password.
+
+
+== Indicator for Lightning mode devices in device lists
+
+Users can easily find devices that support Lightning mode in the device list by:
+
+* Selecting *Lightning mode* in the device search bar and choose *Supported*:
+
+image:lightning-mode-filter-device-list.png[width=400,alt="The Lightning mode filter options in Device List"]
+
+* Choosing *List View*, then *Edit Column*, and check *Lightning mode* to see devices with the Lightning icon:
+
+image:lightning-mode-list-view-column.png[width=250,alt="Show the Lightning mode column in the List View of Device List"]
+
+image:lightning-mode-indicator-device-list.png[width=1000,alt="The lightning indicator on the device under the list view"]
+
+== Script-based automation: obfuscate sensitive data in HTTP headers and Inspector of Session Explorer
+
+Testers who input sensitive data for automated test cases can now use the new custom Appium setting `'kobiton:privateMode'` to hide the data in *Session Explorer*.
+
+If a test step or action happens while the value of `'kobiton:privateMode'` is `true`, the details of it will be hidden in the *View HTTP Headers* and *Appium Inspector* menu of *Session Explorer*.
+
+Below are sample screens with obfuscated data for:
+
+* View HTTP Headers
+
+image:session-explorer-obfuscate-private-data-http-headers.png[width=500,alt="The obfuscated data in the View HTTP Headers section"]
+
+* Appium Inspector
+
+image:session-explorer-obfuscate-private-data-inspector.png[width=500,alt="The obfuscated data in the Appium Inspector section"]
+
+
+== Auto-upgrade deviceConnect: force offline devices if deviceConnect is 2 major versions behind the latest
+
+To ensure Cloud and Hybrid customers can always enjoy the latest features, we added a new policy to force devices offline on the Kobiton portal if deviceConnect on the host machine is 2 major versions behind the latest.
+
+This means for the v4.14 release, all devices hosted by deviceConnect v4.12 or below will become offline. Devices hosted by deviceConnect v4.13 are still online until v4.15 is released (if not updated).
+
+[WARNING]
+There is a known issue that if a new hosting machine with deviceConnect v4.12 or below is added to an organization , the hosting machine and all its devices are not displayed in the Device Management page in the portal. Therefore, make sure you *install deviceConnect v4.13 or later* to new hosting machines.
```

Comment:
Slight revision: When adding a new hosting machine with deviceConnect v4.12 or earlier to an organization, neither the machine nor its devices appear on the Device Management page in the portal. To avoid this issue, ensure that deviceConnect v4.13 or later is installed on all new hosting machines.

Reply:
updated